batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/services/StudentService.java

```diff
@@ -28,8 +36,13 @@ public Student fetchById(int id) throws DataException {
     return stdDAO.fetchById(id);
   }
 
-  public Student edit(Student student) throws DataException {
-    return stdDAO.edit(student);
+  public HashMap edit(Student student) throws DataException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The return type of this method should be an interface such as "Map" rather than the implementation "HashMap". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1319)